projects/1.21/assets/wetland-whimsy/wetland_whimsy/lang/zh_cn.json

```
@@ -0,0 +1,88 @@
+{
+  "advancements.wetland_whimsy.ancient_brazier.desc": "点燃远古火盆",
+  "advancements.wetland_whimsy.ancient_brazier.title": "🔥🔥🔥",
+  "advancements.wetland_whimsy.ancient_pot.desc": "将一枚远古硬币放入远古陶罐",
+  "advancements.wetland_whimsy.ancient_pot.title": "基本薪资",
+  "advancements.wetland_whimsy.dagger.desc": "制作一把伤害随距离变化的匕首",
+  "advancements.wetland_whimsy.dagger.title": "刺杀行动",
+  "advancements.wetland_whimsy.high_quality.desc": "破坏一个装有20枚及以上远古硬币的远古陶罐",
+  "advancements.wetland_whimsy.high_quality.title": "高质量",
+  "advancements.wetland_whimsy.incense.desc": "在远古火盆上使用任意一种熏香",
+  "advancements.wetland_whimsy.incense.title": "你今天好香啊",
+  "advancements.wetland_whimsy.swamp_dungeon.desc": "找到并进入一座沼泽地牢",
+  "advancements.wetland_whimsy.swamp_dungeon.title": "垂直进攻",
+  "advancements.wetland_whimsy.wetland_ruins.desc": "在沼泽群系中找到湿地遗迹",
+  "advancements.wetland_whimsy.wetland_ruins.title": "复古风",
+  "biome.wetland_whimsy.marsh": "草沼",
+  "block.wetland_whimsy.ancient_brazier": "远古火盆",
+  "block.wetland_whimsy.ancient_pot": "远古陶罐",
+  "block.wetland_whimsy.aria_mushroom": "紫晶蘑菇",
+  "block.wetland_whimsy.aria_mushroom_block": "紫晶蘑菇方块",
+  "block.wetland_whimsy.aria_spores": "紫晶孢子",
+  "block.wetland_whimsy.bald_cypress_button": "落羽杉按钮",
+  "block.wetland_whimsy.bald_cypress_cabinet": "落羽杉厨柜",
+  "block.wetland_whimsy.bald_cypress_door": "落羽杉门",
+  "block.wetland_whimsy.bald_cypress_fence": "落羽杉栅栏",
+  "block.wetland_whimsy.bald_cypress_fence_gate": "落羽杉栅栏门",
+  "block.wetland_whimsy.bald_cypress_hanging_sign": "悬挂式落羽杉告示牌",
+  "block.wetland_whimsy.bald_cypress_leaves": "落羽杉树叶",
+  "block.wetland_whimsy.bald_cypress_log": "落羽杉原木",
+  "block.wetland_whimsy.bald_cypress_planks": "落羽杉木板",
+  "block.wetland_whimsy.bald_cypress_pressure_plate": "落羽杉压力板",
+  "block.wetland_whimsy.bald_cypress_sapling": "落羽杉树苗",
+  "block.wetland_whimsy.bald_cypress_sign": "落羽杉告示牌",
+  "block.wetland_whimsy.bald_cypress_slab": "落羽杉台阶",
+  "block.wetland_whimsy.bald_cypress_stairs": "落羽杉楼梯",
+  "block.wetland_whimsy.bald_cypress_trapdoor": "落羽杉活板门",
+  "block.wetland_whimsy.bald_cypress_wood": "落羽杉木",
+  "block.wetland_whimsy.bloodcap_mushroom": "血帽菇",
+  "block.wetland_whimsy.cordgrass": "灯芯草",
+  "block.wetland_whimsy.cordgrass_thatch": "灯芯草捆",
```

```suggestion
  "block.wetland_whimsy.cordgrass": "米草",
  "block.wetland_whimsy.cordgrass_thatch": "米草捆",
```
https://en.wikipedia.org/wiki/Spartina
https://www.iplant.cn/info/Spartina